src/test/java/com/google/api/codegen/testdata/discoveries/nodejs/nodejs_logging.v2beta1.json.baseline

```diff
@@ -40,25 +40,27 @@ google.auth.getApplicationDefault(function(err, authClient) {
     auth: authClient
   };
 
-  var recur = function(err, response) {
+  logging.entries.list(request, function handlePage(err, response) {
```

Comment:
Does `function handlePage` pollute the global namespace?
If I remember correctly, `function foo` puts foo into global namespace where `var foo = function` puts foo in the smallest scope like other variable declarations.
In the sample, I don't think it greatly matters either way though.